builtin/providers/aws/resource_aws_rds_cluster_instance.go

```diff
@@ -83,6 +83,20 @@ func resourceAwsRDSClusterInstance() *schema.Resource {
 				Computed: true,
 			},
 
+			"kms_key_id": &schema.Schema{
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
+				ForceNew: true,
+			},
+
+			"storage_encrypted": &schema.Schema{
```

Comment:
argh, `storage_encrypted` is causing a failure in the import test  https://travis-ci.org/hashicorp/terraform/jobs/146282500